src/main/kotlin/com/zepben/evolve/database/sqlite/writers/MeasurementServiceWriter.kt

```diff
@@ -7,6 +7,5 @@
  */
 package com.zepben.evolve.database.sqlite.writers
 
-class MeasurementServiceWriter(hasCommon: (String) -> Boolean, addCommon: (String) -> Boolean) {
-
-}
+// TODO find out if this is still needed
+class MeasurementServiceWriter(hasCommon: (String) -> Boolean, addCommon: (String) -> Boolean)
```

Comment:
The measurement service doesn't seem to have a corresponding consumer client or consumer service in ewb-network-routes. Do we still need it?

Reply:
I removed it. A proper implementation can be added once we support the measurement service in the database.